maintenance/wikia/email.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * Email
+ *
+ * This sends an email type (defined in extensions/wikia/Email) to the email address of your
+ * choice
+ *
+ */
+
+ini_set( 'display_errors', 'stderr' );
+ini_set( 'error_reporting', E_NOTICE );
```

Comment:
Yes.  I do everything in my power to make PHP behave like a normal language on the command line, which means outputting errors, all errors, to STDERR rather than sending them to the log or silently eating them.  This STILL fails however, if I miss a semicolon.  Apparently that error is just too difficult to recover from and give any output ... >:(